### ISSUE TITLE
[2.0.0-beta1] Calendar view broken with htaccess.RewriteBase => "/"

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/calendar#1555]
Running 2.0.0-beta1 on 17.0.0 doesn't show the calendar view:

![image](https://user-images.githubusercontent.com/3404133/67262618-c4cc1100-f4a5-11e9-9772-21f0988672b1.png)

```
Error: The FullCalendar view "apps" does not exist. Make sure your plugins are loaded correctly.
    at main.esm.js:5635
    at br (main.esm.js:5597)
    at e.reduce (main.esm.js:6733)
    at e.dispatch (main.esm.js:6691)
    at main.esm.js:6664
    at e.batchRendering (main.esm.js:6754)
    at e.hydrate (main.esm.js:6661)
    at new e (main.esm.js:6563)
    at a.mounted (main.esm.js:265)
    at Ge (vue.runtime.esm.js:1854)
He @ vue.runtime.esm.js:1888
Ue @ vue.runtime.esm.js:1879
Ve @ vue.runtime.esm.js:1839
Ge @ vue.runtime.esm.js:1862
tn @ vue.runtime.esm.js:4213
insert @ vue.runtime.esm.js:3139
O @ vue.runtime.esm.js:6340
(anonymous) @ vue.runtime.esm.js:6559
e._update @ vue.runtime.esm.js:3939
r @ vue.runtime.esm.js:4060
hn.get @ vue.runtime.esm.js:4473
hn @ vue.runtime.esm.js:4462
e @ vue.runtime.esm.js:4067
On.$mount @ vue.runtime.esm.js:8409
e._init @ vue.runtime.esm.js:5012
On @ vue.runtime.esm.js:5079
(anonymous) @ main.js:64
i @ bootstrap:63
(anonymous) @ bootstrap:198
(anonymous) @ calendar.js?v=0e849e93-402:1
```

```
vue.runtime.esm.js:1888 TypeError: Cannot read property 'mutateOptions' of undefined
    at a.renderDirty (main.esm.js:342)
    at a.beforeUpdate (main.esm.js:269)
    at Ge (vue.runtime.esm.js:1854)
    at tn (vue.runtime.esm.js:4213)
    at hn.before (vue.runtime.esm.js:4070)
    at fn (vue.runtime.esm.js:4300)
    at Array.<anonymous> (vue.runtime.esm.js:1980)
    at qe (vue.runtime.esm.js:1906)
```

**Comment by juliushaertl** [comment]
Just figured out it does work if I do not use a rewritten url, so 

:boom: /apps/calendar/dayGridMonth/now
:heavy_check_mark: /index.php/apps/calendar/dayGridMonth/now

**Comment by georgehrke** [comment]
Probably related to this line I assume: https://github.com/nextcloud/calendar/blob/master/src/router.js#L36

**Comment by tcitworld** [comment]
Very probably https://github.com/nextcloud/contacts/blob/master/src/router/index.js#L34

**Comment by georgehrke** [comment]
Fix in #1556

**Comment by georgehrke** [comment]
Fixed in #1556
